tests/transform/test_immunohisto_transform.py

```diff
@@ -4,8 +4,13 @@
 import pytest
```

Comment:
I'll review the tests after you simplify. You might be able to remove some of the helper functions and delete/simplify those tests.